src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -141,7 +145,46 @@ def docker_login_to_registry(login_server):
             "Access to registry was denied. Response code: {}".format(response.status_code))
 
     refresh_token = loads(response.content.decode("utf-8"))["refresh_token"]
+    if only_refresh_token:
+        return refresh_token, ''
 
+    authhost = urlunparse((authurl[0], authurl[1], '/oauth2/token', '', '', ''))
+    if repository is None:
+        scope = 'registry:catalog:*'
+    else:
+        scope = 'repository:' + repository + ':pull'
```

Comment:
Correct. When we request an access token, we'll do so with a scope. Since reading the tags for a repository only requires pull privileges then that's what we're asking for. This is the same behavior the docker CLI has. For example, if you want to push an image, the docker CLI will request an access token with the `push` scope on the affected repository.